### ISSUE TITLE
Adding a set webhook to send the message when the bot is online

### ISSUE THREAD

**Issue by Kyro3400** [Khaaz/Cross-Server-Bot#26]
So every time i turn on this bot, it posts the message: `Bot Ready - Cross Server system operational!` But i want to make it so that message won't get posted in every channel that the webhook is in, and i want to set it to a specific channel, but i tried to find a way but not sure, can this be added?

**Comment by Hycord** [comment]
No. The way we hooks works doesn't allow for this. If you want to do this, make the bot send the message. Not the web hooks
